index.html

```diff
@@ -160,7 +160,7 @@ <h1 class="text-sm leading-[1] text-[#555] font-bold">
               <p class="text-gray-500 text-[13px]">Add dates</p>
             </div>
             <div
-              class="rounded text-left p-2 z-[2] shadow-2xl bg-white absolute w-[810px] left-[-250px] top-[70px] rounded-[30px] h-[410px] hidden"
+              class="rounded text-left p-2 z-[2] shadow-2xl bg-white absolute w-[830px] left-[-270px] top-[70px] rounded-[30px] h-[450px] hidden"
```

Comment:
Avoid writing tailwind classes like this ` w-[830px]` use the standard classes if possible

Reply:
okay